lib/saml/user_attributes/ssoe.rb

```diff
@@ -269,9 +269,9 @@ def validate!
       def multiple_id_validations
         # EDIPI, ICN, and CORP ID all trigger errors if multiple unique IDs are found
         check_mhv_icn_mismatch
-        check_id_mismatch(mvi_ids[:vba_corp_ids], :multiple_corp_ids)
+        check_id_mismatch(mvi_ids[:vba_corp_ids], :multiple_corp_ids, mhv_outbound_redirect: mhv_outbound_redirect)
```

Comment:
It looks like we're no longer logging message to sentry when we are `mhv_outbound_redirect` and hit the multiple ids issue. In that case, it might be simpler at this point to refactor these lines to this format:
`check_id_mismatch(mvi_ids[:vba_corp_ids], :multiple_corp_ids) unless mhv_outbound_redirect`
Then we can clean up the `check_id_mismatch` function as well